Source/Libraries/FaultData/DataOperations/ConfigurationOperation.cs

```diff
@@ -1008,49 +999,47 @@ private void AddTriggerChannel(Channel powChannel, MeterDataSet meterDataSet)
                 TableOperations<Channel> channelTable = new TableOperations<Channel>(connection);
                 TableOperations<Series> seriesTable = new TableOperations<Series>(connection);
 
-                trendChannel.MeasurementType = measurementTypeTable.QueryRecordWhere("ID = {0}", trendChannel.MeasurementTypeID);
-                trendChannel.MeasurementCharacteristic = measurementCharacteristicTable.QueryRecordWhere("ID = {0}", trendChannel.MeasurementCharacteristicID);
+                string measCharacteristic = "";
```

Comment:
Since we're changing all this code anyway, I'd prefer if we spelled it out.
```suggestion
                string measurementCharacteristic = "";
```